.travis.yml

```diff
@@ -14,7 +14,7 @@ before_install:
 script:
   # Setup the PR and merge into current master
   - git fetch origin master
-  - git checkout origin/master
+  - git checkout origin/master -b master
```

Comment:
this still errors in Travis

Reply:
It is failed with or without `-b master`